src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/LogsController.php

```diff
@@ -122,7 +122,7 @@ public function saveSettingsAction(Request $request)
                 return $this->redirectToRoute('admin_logs_index');
             }
 
-            $this->flashErrors($saveErrors);
+            $this->addFlash('error', $this->trans('Invalid email address.', 'Admin.Notifications.Error') . ' ( ' . implode(',', $saveErrors) . ' ) ');
```

Comment:
shouldn't we check `$saveErrors` is not empty before calling the `addFlash` method?

Reply:
otherwise lgtm 👍